var/spack/repos/builtin/packages/ascent/package.py

```diff
@@ -85,6 +95,7 @@ class Ascent(Package):
     depends_on("python+shared", when="+python+shared")
     extends("python", when="+python+shared")
     depends_on("py-numpy", when="+python+shared", type=('build', 'run'))
+    depends_on("py-pip", when="+python+shared", type=('build', 'run'))
```

Comment:
pip is rarely needed if you follow the standard `setup.py install` method

Reply:
agree, it looks dubious -- but we need it for some things post spack install (testing and some use cases). It's very convenient so i'd like to keep it.